app-catalog/src/api/catalogs.tsx

```diff
@@ -0,0 +1,53 @@
+import { QueryParameters, request } from '@kinvolk/headlamp-plugin/lib/ApiProxy';
+import { ChartsList, COMMUNITY_REPO, VANILLA_HELM_REPO } from '../components/charts/List';
+
+const SERVICE_ENDPOINT = '/api/v1/services';
+const LABEL_CATALOG = 'catalog.ocne.io/is-catalog';
+
+declare global {
```

Comment:
I'm not sure about using global variables. 

This fills up the global name space so all other headlamp code and other plugin code might conflict with this.

Reply:
This was fixed.